packages/redux/src/wishlists/selectors/wishlistsSets.ts

```diff
@@ -173,7 +178,7 @@ export const getWishlistSet = createSelector(
     return {
       ...wishlistSet,
       wishlistSetItems,
-    };
+    } as WishlistSetHydrated;
```

Comment:
I think this can be `undefined` as well 🤔

Reply:
Done 🙂